Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:alpine
+FROM node:12-alpine
```

Comment:
Note that use node-14 will sometime get an error when building images

eg.

![image](https://user-images.githubusercontent.com/40660121/94456317-1452ab00-01e6-11eb-9818-6f9db3fd25e2.png)